index.html

```diff
@@ -13,10 +13,18 @@
       <div class="left-side-head">
         <a href="#" class="name">Victor Toraño</a>
       </div>
-      <div class="righ-side-head">
-        <a href="#" class="name-desk">Porfolio</a>
-        <a href="#" class="name-desk">About</a>
-        <a href="#" class="name-desk">Contact</a>
+      <div class="Items-display">
+        <img src="/pictures/cross.svg" class="cross" alt="">
```

Comment:
Kindly push the icon more to the left as it is not visible now 
![wrong](https://user-images.githubusercontent.com/58376020/127195178-767e6e7b-5384-459b-925c-9c130a08ed4f.png)